src/cosmicqc/__init__.py

```diff
@@ -3,8 +3,8 @@
 """
 
 from .analyze import find_outliers
-from .frame import SCDataFrame
+from .frame import CytoDataFrame
 
 # note: version placeholder is updated during build
 # by poetry-dynamic-versioning.
-__version__ = "0.0.0"
+__version__ = "0.0.6.post3.dev0+1bfa792"
```

Comment:
I probably asked this question before but does this change automatically each PR?

Reply:
Thanks for asking about this, it's a persisted version from a `poetry-dynamic-versioning` setting. I think this has been happening in error and I made an adjustment to the `pyproject.toml` in an effort to fix this as well.